### DISCUSSION TITLE
删除网站把项目也删除

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#6223] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

如题为什么删除网站要删除代码啊.我只是取错了域名想换个直接删除代码都给删了 没一点点的防备

**Comment by wanghe-fit2cloud** [comment]
如需变更域名，可进入 `网站设置` 中的 `域名设置` 页面进行维护。

**Reply by iygwei** [reply]
我知道 但是有时候想改容器的名或者外面的那个名字 就只能删除,但是一删除文件就跟着删除了

**Reply by wanghe-fit2cloud** [reply]
修改主域名：
![image](https://github.com/user-attachments/assets/27f08409-2e88-4c33-9a33-26197b06fe1f)

**Reply by iygwei** [reply]
重点是不应该删除
